qemu/tests/timedrift_no_net.py

```diff
@@ -148,7 +148,7 @@ def run(test, params, env):
     session.close()
 
     if sub_work in globals():  # Try to find sub work function.
-        globals()[sub_work](params, vm, session)
+        globals()[sub_work](test, params, vm, session)
```

Comment:
Should have the same change for `timedrift_no_net_win.py`, right?

Reply:
Thanks, modified it.